src/screens/Onboarding/StepProfile/PlaceholderCanvas.tsx

```diff
@@ -33,12 +37,12 @@ export const PlaceholderCanvas = React.forwardRef<PlaceholderCanvasRef, {}>(
 
     React.useImperativeHandle(ref, () => ({
       // @ts-ignore this library doesn't have types
-      capture: viewshotRef.current.capture,
+      capture: viewshotRef.current?.capture,
```

Comment:
are we ok with this method being null in some cases? should we instead wrap to make it a noop?

Reply:
i tweaked it like this so it will show up as possibly being undefined [`3b220f6` (#5517)](https://github.com/bluesky-social/social-app/pull/5517/commits/3b220f6e82c108dfa451b65dd16243db5f7453fd)

im going to see if i can just get rid of that ts-ignore though, i think there actually is a type for this

Reply:
okay yea, this feels nicer [`1006d70` (#5517)](https://github.com/bluesky-social/social-app/pull/5517/commits/1006d70ed106fe59f1a026d1bf9180e14868722c)